rest-model/src/main/java/com/gentics/mesh/core/rest/node/field/impl/BinaryFieldImpl.java

```diff
@@ -66,7 +66,7 @@ public String getType() {
 
 	@JsonProperty(required = true)
 	@JsonPropertyDescription("Status of the external binary check (one of ACCEPTED, DENIED or POSTPONED). The binary field is only accessible when the status is ACCEPTED.")
-	private BinaryCheckStatus checkStatus = BinaryCheckStatus.POSTPONED;
+	private BinaryCheckStatus checkStatus = BinaryCheckStatus.ACCEPTED;
```

Comment:
This is not a fix of the problem, it only changes the situation, where the problem occurs.
The real problem is that the request to change a binary field can contain the "checkStatus" field, which is not changeable (and should not be), but Mesh will compare the sent checkStatus (or the default value) with the current checkStatus of the field and if they are different, will create a new version.
The correct solution should be to ignore the (unmodifiable) field "checkStatus" in the request when comparing the fields.